packages/ods/src/components/input/src/components/ods-input/ods-input.scss

```diff
@@ -94,6 +101,7 @@ $ods-input-input-padding-right: calc($ods-input-actions-button-width + $ods-inpu
     &:disabled {
       ~ .ods-input__actions {
         cursor: not-allowed;
+        pointer-events: none;
```

Comment:
would be best to manage this on the button style instead
update the `:active` selector with a `:not(:disabled)`